fix: made community resources page responsive

MauryanKansara opened the pull request:
## Fixes Issue
fixes: #8962 

## Changes proposed

Changed DocsLayout.js file. Added "overflow-auto" in the div where children is rendered so for small devices ( like mobile or tablet ) the "contributions" table looks good and does not cause overflow.

## Screenshots

![Screenshot (185)](https://github.com/EddieHubCommunity/BioDrop/assets/86093112/c4d710a1-b248-4a58-aeca-d4373a3434e5)

amandamartin-dev reviewed (changes requested):
Hello! Thank you for taking a look at this issue.  I think we can make it a little better.  Can you adjust the code to only target the table to scroll on a small viewport?  Currently, it's scrolling up through the area with the code snippets creating unneccesary white space.

In addition, if you can make the section header static so that it does not scroll and only the content that needs to it will help keep the context in view.  I've added a screenshot to show what I mean, please let me know if you have any questions.


![screenshot](https://github.com/EddieHubCommunity/BioDrop/assets/97615019/fc62d63f-ac01-4f72-8f93-5f559a6a7598)

MauryanKansara commented:
I was also thinking the same and was about to debug this problem 😅. I will push new changes very soon!

MauryanKansara commented:
hey @amandamartin-dev I've pushed the new changes as you said and I also modified the font size and line spacing for smaller devices.

amandamartin-dev commented on pages/docs/community-resources.mdx:
```diff
@@ -49,34 +49,36 @@ YouTube (or blog)
-  </tbody>
-</table>
+<div className="w-full overflow-auto">
+  <table className="text-xs leading-5 md:text-base md:leading-none">
```
you can remove all the classes here, we want to keep each page consistent in the way it is styled and thus far we do not include any inline styling at the table level on any docs pages. Otherwise the changes look good! 
```suggestion
  <table>
```

amandamartin-dev reviewed (approved):
Thank you for addressing all the comments and for the great collaboration! Everything looks good. Tested in gitpod.

MauryanKansara commented:
> Thank you for addressing all the comments and for the great collaboration! Everything looks good. Tested in gitpod.

thankss!!! It was my first PR for this repository and gained really good experience😀, can you please merge this PR? so I can start working on new issues 😅

eddiejaoude commented:
Great collaboration 💪 Merging now
